### PR TITLE
Normalize - foursquare

### PR DESCRIPTION
**Pull request by SPTKL**

### CODE REVIEW

**Review by mgraber — CHANGES_REQUESTED** [review]
See updated issue #73:

- [ ] Change table names to something more descriptive (refer to county/zip rather than foursquare/foursquare_datacube)
- [ ] Only use the grouped data -- the only input records should be those where 
```sql 
categoryid = 'Group'
```
- [ ] Remove `state` from foursquare_datacube_grouped
- [ ] Backfilling  a `borough` field and a `borocode` field using the zipcodes
- [ ] Since NYC-only, keep `borough` but remove `county` in foursquare_datacube_grouped (it is empty now)
- [ ] Format `date` as `YYYY-MM-DD` in foursquare and foursquare_grouped

**Comment by mgraber** [comment]
Also see #82

File: recipes/foursquare/create_county.sql [hunk 1]
```diff
@@ -27,62 +28,46 @@ WITH unpivot AS(
...
+    borough,
+    categoryname,
+    SUM(visits_all) AS visits_all,
+    SUM(visits_u65) AS visits_u65,
+    SUM(visits_o65) AS visits_o65,
+    ROUND(SUM(duration_avg_all*visits_all)/SUM(visits_all), 2) as duration_avg_all,
+    ROUND(SUM(duration_avg_u65*visits_u65)/SUM(visits_u65), 2) as duration_avg_u65,
+    ROUND(SUM(duration_avg_o65*visits_o65)/SUM(visits_o65), 2)as duration_avg_o65
+INTO foursquare.daily_county
+FROM :NAME.:"VERSION" WHERE categoryid != 'Group'
```
Comment (AmandaDoyle) [inline recipes/foursquare/create_county.sql]: should this be where categoryid == 'Group'?

**Review by AmandaDoyle — COMMENTED** [review]
Have one question, but no additional comments

File: recipes/foursquare/runner.sh [hunk 2]
```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 source $(pwd)/bin/config.sh
-source $(pwd)/recipes/foursquare/runner_datacube.sh
+source $(pwd)/recipes/foursquare/runner_zipcode.sh
 source $(pwd)/recipes/foursquare/runner_county.sh
-TYPE=$1 # datacube/*
+TYPE=$1 # datacube/county
```
Comment (mgraber) [inline recipes/foursquare/runner.sh:5]: `# zipcode/county`

File: recipes/foursquare/init.sql [hunk 3]
```diff
@@ -40,100 +32,133 @@ BEGIN
...
+    ELSE RAISE NOTICE 'foursquare_zipcode.daily_zipcode is created';
     END IF;
 
-    IF NOT _view_grouped_weekly_zipcode THEN
-        CREATE VIEW foursquare_datacube.grouped_weekly_zipcode AS (
+    IF NOT _view_weekly_zipcode THEN
+        CREATE VIEW foursquare_zipcode.weekly_zipcode AS (
             SELECT
                 to_char(date::date, 'IYYY-IW') as year_week,
                 zip as zipcode,
```
Comment (mgraber) [inline recipes/foursquare/init.sql:108]: Add borough and borocode to foursquare.weekly_zipcode

**Review by mgraber — CHANGES_REQUESTED** [review]

**Review by mgraber — APPROVED** [review]

**Pull request merged**